src/resources/views/crud/form_content.blade.php

```diff
@@ -5,7 +5,7 @@
     @include('crud::inc.show_tabbed_fields')
     <input type="hidden" name="_current_tab" value="{{ Str::slug($crud->getTabs()[0]) }}" />
 @else
-  <div class="card">
+  <div class="card border-0">
     <div class="card-body row">
       @include('crud::inc.show_fields', ['fields' => $crud->fields()])
     </div>
```

Comment:
Icon picker field is broken. But that's a `v6` problem, not a `coreuiv4` problem. Once it's fixed there, I will pull the changes here too. https://github.com/Laravel-Backpack/PRO/pull/123/files#r1111788207

Reply:
Fixed on `v6`. I just pulled the fix to this branch too.